interface/billing/billing_report.php

```diff
@@ -52,18 +52,15 @@
     }
 
     if ($_POST['mode'] == 'export') {
-        $sql = ReturnOFXSql();
-        $db = get_db();
-        $results = $db->Execute($sql);
+        $results = ReturnOFXSql();
         $billings = array();
-        if ($results->RecordCount() == 0) {
+        if (sizeof($results) == 0) {
```

Comment:
Note sqlStatement is not returning an array, but a recordset. So for above would do:
```
if (sqlNumRows($results) == 0) {
```
this is a function in library/sql.inc

Reply:
```suggestion
        if (sqlNumRows($results) == 0) {
```